packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/UserInfoDetails/UserInfoDetails.template.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+// @ts-ignore
+import { defaultTo, replace } from 'ramda'
+import { Field, InjectedFormProps, reduxForm } from 'redux-form'
+
+import { CountryScope } from '@core/types'
+import { BlockchainLoader, Button, HeartbeatLoader, Icon, Text } from 'blockchain-info-components'
+import {
+  FlyoutContainer,
+  FlyoutContent,
+  FlyoutFooter,
+  FlyoutHeader
+} from 'components/Flyout/Layout'
+import DateInputBox from 'components/Form/DateInputBox'
+import FormGroup from 'components/Form/FormGroup'
+import FormItem from 'components/Form/FormItem'
+import TextBox from 'components/Form/TextBox'
+import { model } from 'data'
+import { useCountryList, useUSStateList } from 'hooks'
+import { ageOverEighteen, required, requiredDOB } from 'services/forms'
+
+import { Props as OwnProps, SuccessStateType } from './UserInfoDetails'
+import {
+  Caption,
+  CaptionContainer,
+  ContentWrapper,
+  CustomForm,
+  ErrorText,
+  ErrorTextContainer,
+  Label,
+  SpinnerWrapper
+} from './UserInfoDetails.model'
+
+const { USER_INFO_DETAILS } = model.components.identityVerification
+
+const addTrailingZero = (text: string) => (text.length >= 2 ? text : `0${text}`)
+
+const removeTrailingZero = replace(/^0/, '')
+
+const objectToDOB = ({ date = '', month = '', year = '' }) =>
+  `${year}-${month}-${addTrailingZero(date)}`
+
+const DOBToObject = (value) => {
+  const [year = '', month = '', date = ''] = defaultTo('', value).split('-')
+  return {
+    date: removeTrailingZero(date),
+    month,
+    year
+  }
+}
+
+const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
```

Comment:
`props` could be destructured either in the parameter definition, or in the first lines in the function - that would reduce the `props. ... props.` throughout the entire function